### PR TITLE
feat: update javascript example with send_object_to parameter

### PR DESCRIPTION
**Pull request by yanganto**

### CODE REVIEW

File: examples/javascript/blob_upload_download_webapi.html [hunk 1]
```diff
@@ -241,6 +243,15 @@ <h2>Blob Upload</h2>
               </div>
             </div>
 
+            <div class="col-12">
+              <label for="send-to-input" class="form-label">Send to</label>
+              <input id="send-to-input" value="0x9d8a07f9f5cb4d10ad3a8c949f17b9ae6c51fc76c2b7ef62dc5babec551e79e6" placeholder="Send to" class="form-control"
+                required />
+              <div class="form-text">
+                The number of Walrus epochs for which to store the blob.
```
Comment (mlegner) [inline examples/javascript/blob_upload_download_webapi.html:251]: This needs to be updated:
```suggestion
                The address to which the newly created `Blob` object should be sent (optional).
```

File: examples/javascript/blob_upload_download_webapi.html [hunk 2]
```diff
@@ -241,6 +243,15 @@ <h2>Blob Upload</h2>
               </div>
             </div>
 
+            <div class="col-12">
+              <label for="send-to-input" class="form-label">Send to</label>
+              <input id="send-to-input" value="0x9d8a07f9f5cb4d10ad3a8c949f17b9ae6c51fc76c2b7ef62dc5babec551e79e6" placeholder="Send to" class="form-control"
```
Comment (mlegner) [inline examples/javascript/blob_upload_download_webapi.html:248]: I think this shouldn't have a value by default.
```suggestion
              <input id="send-to-input" placeholder="0x9d8a07f9f5cb4d10ad3a8c949f17b9ae6c51fc76c2b7ef62dc5babec551e79e6" class="form-control"
```

**Review by mlegner — APPROVED** [review]
Thanks for the contribution, @yanganto! Two comments from my side.

**Comment by yanganto** [comment]
@mlegner Thank you. :heart:

**Pull request merged**